websites/N/Navidrome/presence.ts

```diff
@@ -0,0 +1,42 @@
+const presence = new Presence({
+	clientId: "1126923786600583339",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/mhD2eIr.png",
+	};
+
+	// Determine player state
+	let playerState;
+	// Check if audio-title has any content
+	if (document.querySelectorAll(".audio-title")[0].innerHTML !== "") {
+		if (document.querySelectorAll(".react-jinke-music-player-pause-icon")[0])
+			playerState = "playing";
+		else playerState = "paused";
+	} else playerState = "notInitialized";
+
+	// Check if music-player-panel initialized || Check if any music is playing/paused
+	if (playerState === "playing" || playerState === "paused") {
+		// Grab song information
+
+		const timestamps = presence.getTimestamps(
+			presence.timestampFromFormat(
+				document.querySelectorAll(".current-time")[0].innerHTML
```

Comment:
If u get the first one from querySelectorAll then just use querySelector
```suggestion
				document.querySelectol(".current-time").innerHTML
```